### ISSUE TITLE
BUG: 

### ISSUE THREAD

**Issue by brainlykcz** [neptune-ai/neptune-client#627]
### Describe the bug
hello world does not work - it produces 
`TypeError: 'Project' object does not support item assignment`

### Reproduction

```
import neptune

neptune_run = neptune.init(
    project_qualified_name="workspace/project",
    api_token="********",
)
neptune_run["parameters"] = dict(a=1, b=2)

```

### Expected behavior
no error 
### Traceback

```
Traceback (most recent call last):
  File "hello.py", line 8, in <module>
    neptune_run["parameters"] = dict(a=1, b=2)
```

### Environment
**The output of `pip list`:**
```
Package                  Version
------------------------ -----------
absl-py                  0.12.0
addict                   2.4.0
aiobotocore              1.3.0
aiohttp                  3.7.4.post0
aioitertools             0.7.1
altair                   4.1.0
anyio                    2.2.0
appdirs                  1.4.4
argon2-cffi              20.1.0
astor                    0.8.1
astunparse               1.6.3
async-generator          1.10
async-timeout            3.0.1
attrs                    20.3.0
Babel                    2.9.1
backcall                 0.2.0
base58                   2.1.0
black                    20.8b1
bleach                   3.3.0
blinker                  1.4
boto3                    1.17.49
botocore                 1.20.49
bravado                  11.0.3
bravado-core             5.17.0
cachetools               4.2.2
certifi                  2020.12.5
cffi                     1.14.5
chardet                  4.0.0
click                    7.1.2
configparser             5.0.2
cycler                   0.10.0
Cython                   0.29.23
decorator                5.0.7
defusedxml               0.7.1
deprecation              2.1.0
deskew                   0.10.28
dill                     0.3.3
docker-pycreds           0.4.0
eli5                     0.11.0
entrypoints              0.3
enum-compat              0.0.3
flatbuffers              1.12
fsspec                   2021.4.0
future                   0.18.2
gast                     0.3.3
gitdb                    4.0.7
GitPython                3.1.15
google-auth              1.30.0
google-auth-oauthlib     0.4.4
google-pasta             0.2.0
graphviz                 0.16
grpcio                   1.32.0
h5py                     2.10.0
horology                 1.1.0
idna                     2.10
imageio                  2.9.0
importlib-metadata       4.0.1
iniconfig                1.1.1
ipykernel                5.5.4
ipython                  7.23.1
ipython-genutils         0.2.0
ipywidgets               7.6.3
jedi                     0.18.0
Jinja2                   2.11.3
jmespath                 0.10.0
joblib                   1.0.1
json5                    0.9.5
jsonpointer              2.1
jsonref                  0.2
jsonschema               3.2.0
jupyter-client           6.2.0
jupyter-core             4.7.1
jupyter-packaging        0.9.2
jupyter-server           1.6.4
jupyterlab               3.0.14
jupyterlab-pygments      0.1.2
jupyterlab-server        2.5.0
jupyterlab-widgets       1.0.0
Keras-Preprocessing      1.1.2
kiwisolver               1.3.1
lime                     0.2.0.1
Markdown                 3.3.4
MarkupSafe               1.1.1
matplotlib               3.3.4
matplotlib-inline        0.1.2
mean-average-precision   2021.4.26.0
mistune                  0.8.4
mmcv                     1.3.9
mmdet                    2.14.0
monotonic                1.6
msgpack                  1.0.2
multidict                5.1.0
multiprocess             0.70.11.1
mypy                     0.812
mypy-extensions          0.4.3
nbclassic                0.2.7
nbclient                 0.5.3
nbconvert                6.0.7
nbformat                 5.1.3
neptune-client           0.10.1
neptune-tensorflow-keras 0.9.1
nest-asyncio             1.5.1
networkx                 2.5
notebook                 6.3.0
numpy                    1.18.5
oauthlib                 3.1.0
opencv-python            4.5.1.48
opt-einsum               3.3.0
packaging                20.9
pandas                   1.2.4
pandocfilters            1.4.3
parso                    0.8.2
pathos                   0.2.7
pathspec                 0.8.1
pathtools                0.1.2
pexpect                  4.8.0
pickleshare              0.7.5
pigeon-jupyter           0.1.0
Pillow                   8.2.0
pip                      21.0.1
plotly                   4.14.3
pluggy                   0.13.1
pox                      0.2.9
ppft                     1.6.6.3
prometheus-client        0.10.1
promise                  2.3
prompt-toolkit           3.0.18
protobuf                 3.15.8
protobuf3-to-dict        0.1.5
psutil                   5.8.0
ptyprocess               0.7.0
py                       1.10.0
pyarrow                  4.0.0
pyasn1                   0.4.8
pyasn1-modules           0.2.8
pycocotools              2.0.2
pycparser                2.20
pydeck                   0.6.2
Pygments                 2.9.0
PyJWT                    2.1.0
pyparsing                2.4.7
pyrsistent               0.17.3
pytest                   6.2.2
python-dateutil          2.8.1
python-dotenv            0.15.0
pytz                     2021.1
PyWavelets               1.1.1
PyYAML                   5.4.1
pyzmq                    22.0.3
regex                    2021.4.4
requests                 2.25.1
requests-oauthlib        1.3.0
retrying                 1.3.3
rfc3987                  1.3.8
rsa                      4.7.2
s3fs                     0.6.0
s3transfer               0.3.7
sagemaker                2.33.0
sagemaker-experiments    0.1.30
scikit-image             0.18.1
scikit-learn             0.24.2
scipy                    1.4.1
Send2Trash               1.5.0
sentry-sdk               1.0.0
setuptools               52.0.0
Shapely                  1.7.1
shortuuid                1.0.1
simplejson               3.17.2
six                      1.15.0
smdebug-rulesconfig      1.0.1
smmap                    4.0.0
sniffio                  1.2.0
streamlit                0.81.1
strict-rfc3339           0.7
subprocess32             3.5.4
swagger-spec-validator   2.7.3
tabulate                 0.8.9
tensorboard              2.5.0
tensorboard-data-server  0.6.1
tensorboard-plugin-wit   1.8.0
tensorflow               2.3.0
tensorflow-addons        0.13.0
tensorflow-estimator     2.3.0
tensorflow-hub           0.11.0
termcolor                1.1.0
terminado                0.9.4
terminaltables           3.1.0
testpath                 0.4.4
threadpoolctl            2.1.0
tifffile                 2021.4.8
toml                     0.10.2
tomlkit                  0.7.0
toolz                    0.11.1
torch                    1.9.0
torch-model-archiver     0.4.0
torchserve               0.4.0
torchvision              0.10.0
tornado                  6.1
tqdm                     4.60.0
traitlets                5.0.5
typed-ast                1.4.3
typeguard                2.12.0
typing-extensions        3.7.4.3
tzlocal                  2.1
urllib3                  1.26.4
validators               0.18.2
wandb                    0.10.29
watchdog                 2.1.0
wcwidth                  0.2.5
webcolors                1.11.1
webencodings             0.5.1
websocket-client         1.0.1
Werkzeug                 1.0.1
wheel                    0.36.2
widgetsnbextension       3.5.1
wrapt                    1.12.1
yapf                     0.31.0
yarl                     1.6.3
zipp                     3.4.1

```
ubuntu 18.04 
**The output of `python --version`: Python 3.8.6 **
### Additional context
Add any other context about the problem here: https://docs.neptune.ai/you-should-know/logging-metadata

**Comment by Blaizzy** [comment]
Hi @brainlykcz,

I'm Prince Canuma, DevRel at Neptune.ai

I notice that you used `project_qualified_name`  and you imported `neptune`  can you mention where in the docs you found this example?

**Comment by Blaizzy** [comment]
Try this way
```
import neptune.new as neptune

neptune_run = neptune.init(
    project="workspace/project",
    api_token="********",
)
neptune_run["parameters"] = dict(a=1, b=2)

```

Currently, if you want to use our latest client you have to do `import neptune.new as neptune`.
We are in a transition to the new client so if you do `import neptune` you are calling the old client.


And `neptune.init()` uses `project` argument, you can find more about it here:
https://docs.neptune.ai/api-reference/neptune#init

**Comment by brainlykcz** [comment]
thank you Prince - I missed `.new` :)

**Comment by Blaizzy** [comment]
Great, most welcome!

Can you please let me know where you copy this code:

```
import neptune

neptune_run = neptune.init(
    project_qualified_name="workspace/project",
    api_token="********",
)
neptune_run["parameters"] = dict(a=1, b=2)

```

**Comment by brainlykcz** [comment]
I didn't copy it fortunately for you - unfortunately for me - it's a product of my caffeine-less morning head ;) 

have a great day!

**Comment by Blaizzy** [comment]
Super 💯 

I hope you have a great day too!
